experiment/autobumper/bumper/bumper.go

```diff
@@ -46,6 +48,39 @@ func Call(cmd string, args ...string) error {
 	return c.Run()
 }
 
+func call(stdout, stderr io.Writer, cmd string, args ...string) error {
+	c := exec.Command(cmd, args...)
+	c.Stdout = stdout
+	c.Stderr = stderr
+	return c.Run()
+}
+
+const (
+	censored = "CENSORED"
+)
+
+var censoredBytes = []byte(censored)
+
+type hideSecretsWriter struct {
+	secretsMap map[string][]byte
+	delegate   io.Writer
+}
+
+func (w hideSecretsWriter) Write(content []byte) (int, error) {
```

Comment:
I think I'd rather see the tool load the secret agent?

Reply:
+1. Is there any reason why we can't use the `git` package from Prow for the wrapped git calls?

Reply:
Technically the `git` package is not meant to be a all-purpose `git` client :|

Reply:
> I think I'd rather see the tool load the secret agent?

Clarified via slack.